custom/conf/app.ini.sample

```diff
@@ -265,9 +265,9 @@ COOKIE_REMEMBER_NAME = gitea_incredible
 REVERSE_PROXY_AUTHENTICATION_USER = X-WEBAUTH-USER
 ; The minimum password length for new Users
 MIN_PASSWORD_LENGTH = 6
-; True when users are allowed to import local server paths
+; Set to true when users are allowed to import local server paths
 IMPORT_LOCAL_PATHS = false
-; Prevent all users (including admin) from creating custom git hooks
+; Set to false to prevent all users (including admin) from creating custom git hooks
```

Comment:
This is backwards(?) This should say: 'Set to true to prevent…'. This is the exact reason I think the docs were (and possibly still are) confusing. The setting names are boolean opposites and make it difficult to reason about. Not ideal for security settings.

Reply:
You're right. Thanks for catching that.